autoyast/provision.erb

```diff
@@ -113,6 +115,9 @@ cp /etc/resolv.conf /mnt/etc
         <notification>Setting up Puppet / Foreman ...</notification>
         <source><![CDATA[
 /bin/hostname <%= @host.name %>
+
+<%= snippet('remote_execution_ssh_keys') %>
+
 <% if puppet_enabled -%>
```

Comment:
We don't support chef on SUSE at the moment (I also did no research if and how that would be possible)